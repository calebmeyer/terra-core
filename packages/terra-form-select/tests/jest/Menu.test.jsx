import React from 'react';
import Menu from '../../src/_Menu';
import intlContexts from './intl-context-setup';

describe('Menu', () => {
  it('should render a default Menu', () => {
    const liveRegion = { current: document.createElement('div') };

    const menu = <Menu onSelect={() => {}} visuallyHiddenComponent={liveRegion} variant="default" value="value" />;
    const wrapper = shallow(menu, intlContexts.shallowContext);
    expect(wrapper).toMatchSnapshot();
  });
});
