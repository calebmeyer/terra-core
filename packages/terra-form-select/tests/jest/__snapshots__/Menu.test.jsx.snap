// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu should render a default Menu 1`] = `
<ul
  class="menu"
  data-activedescendant="terra-select-option-null"
  id="terra-select-menu"
  role="listbox"
  style="outline:none"
  tabindex="0"
>
  <div
    class="no-results"
  >
    No matching results for ""
  </div>
</ul>
`;
